framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -962,8 +953,9 @@ private void message(
         try {
             messageText = String.format(fmtString, args);
         } catch (Exception e) {
-            messageText =
-                    "Invalid format string: \"" + fmtString + "\" args: " + Arrays.toString(args);
+            throw new BugInCF(
```

Comment:
This change looks good and will indeed help us catch format string errors much easier.